### DISCUSSION TITLE
smartthings e nabu casa

### DISCUSSION THREAD

**Discussion by tibur79** [indomus/forum#2572] [GENERALE - Discussioni generiche]
giorno ho questo problema con smartthings non riusciendolo a configurare ricevo errore 404, ma ho nabu casa e il log è il seguente:

API error setting up the SmartApp: {'requestId': '1041211565366462805', 'error': {'code': 'ConstraintViolationError', 'message': 'The request is malformed.', 'details': [{'code': 'TargetNon200Error', 'target': 'https://hooks.nabu.casa/gAAAAABkw-a4tOh2AYR0WFTPNdQgbxN7tfU2mmc4f130ePsiTtqLHyuAz1_69HRXSrvLrMHDFtqF0IjMhC-tXVDVtJYIy_YJES6okZWr9j57pez6iA0Q67FlEGsxpvgeNdiAMvVG5fItfqzlbOPIgMxNacrmlbUk-vqQmDpRFjfYQjIHDyBsK8A=', 'message': 'Target did not respond with a 200 status. Upstream status code=404', 'details': []}]}}

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/smartthings/config_flow.py", line 126, in async_step_pat
    app, client = await create_app(self.hass, self.api)

**Comment by snakuzzo** [comment]
L'errore è chiaro...
se fai una GET verso `https://hooks.nabu.casa/gAAAAABkw-a4tOh2AYR0WFTPNdQgbxN7tfU2mmc4f130ePsiTtqLHyuAz1_69HRXSrvLrMHDFtqF0IjMhC-tXVDVtJYIy_YJES6okZWr9j57pez6iA0Q67FlEGsxpvgeNdiAMvVG5fItfqzlbOPIgMxNacrmlbUk-vqQmDpRFjfYQjIHDyBsK8A=` ti restituisce http 404.

Ricontrolla tutto il giro relativo all'esposizione di HA con Nabucasa.

**Reply by tibur79** [reply]
grazie per la risposta ma cosa devo inserire l'indirro ip del mio ha?

**Reply by snakuzzo** [reply]
Ma dove ?
Hai capito quello che ti ho detto ?

Leggi e segui le istruzioni indicate [QUI](https://www.home-assistant.io/integrations/smartthings/)  e [QUI](https://www.nabucasa.com/config/webhooks/)

**Reply by tibur79** [reply]
scusa ma non sono così efferato chiedo aiuto apposta riesco a vedere il mio indirizzo di nabu casa tramite telecomando nel cloud ma non so quale sia quello giusto da inserire poi nelle modifiche in 4 g accedo tranquillamente

**Reply by snakuzzo** [reply]
Ok, ma devi seguire le indicazioni per procedere con l'integrazione SmartThings. Nel mio post precedente trovi i due link.

**Reply by tibur79** [reply]
sera scusami ma le ho provate tutte e ricevo lo stesso messaggio se provo il mio url in 4G riesco ad accedere ad HA senza problemi non capisco cosa dovrei cambiare nella mia configurazione c'è altro modo per controllare i dispositivi samsung ?
